summer_project/coc.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Indian Linux User Group Delhi</title>
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans:400,700" rel="stylesheet">
+    <!--<link href="https://fonts.googleapis.com/css?family=Space+Mono" rel="stylesheet">-->
+    <link rel="stylesheet" type="text/css" href="../assets/css/main.css">
+</head>
+<a href="https://github.com/ILUGD/ILUGD.github.io" target = "_blank"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/38ef81f8aca64bb9a64448d0d70f1308ef5341ab/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6461726b626c75655f3132313632312e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png"></a>
+<body class="page-home">
+  <section>
+      <h1>Code of conduct</h1><br>
+      
+      <div align="left">
+        
+        <p><b>General guideline: </b>
+        Contributors and maintainers should participation in our project 
+        and make our community a harassment-free experience for everyone, regardless of age, 
```

Comment:
Need to rephrase the line, `Contributors and maintainers should participation in our project` doesn't make any sense